README.md

```diff
@@ -1 +1,49 @@
-# Hello World
+# Github Governance
+
+This readme is for building out our github gonvernance.
+
+## Top level SWRCB GitHub Documentation that covers our guiding principals, protocols, management decisions, and policies
+Also, this is open to discussion. 
+
+### Things that might go in this documnet or in this repo:
+1. Data Governanace guiding pricipals and documents
+	1. A code of conduct [possibily this one](https://github.com/skills/.github/blob/main/CODE_OF_CONDUCT.md)
+	2. Open data policy documentation
+		1. CHHS has a [Open Data Handbook](https://chhsdata.github.io/opendatahandbook/) we might considering forking or adopting in all or part. [Their githubpage](https://github.com/chhsdata/opendatahandbook)
+	3. [F(findable) A(accessible) I(interoperable) R(reusable)](https://www.go-fair.org/fair-principles/)
+2. Github account management, repo administration, 
+	1. Default Account permissions for WaterBoard members and outside collaborators
+	2. You can define which individuals or teams own certain types or areas of code in a CODEOWNERS file.
+	3. Other ideas to implement
+		1. Contributing guidlines for internal/external collaborators [possibly this one](https://github.com/skills/.github/blob/main/CONTRIBUTING.md)
+		2. Justification for creation of Repos
+		3. Actions to be taken after the creation of repos.
+2. GitHub Wiki
+	1.  [NOAA NMFS's Openscapes org](https://github.com/nmfs-openscapes/.github/wiki) - the gathering place for NOAA open science use rather than the actual guidelines (e.g., see the different [licenses used](https://github.com/nmfs-openscapes/.github/wiki/05B-Licenses-for-government-work))
+	2. [Other guidelines - NOAA](https://noaa-fisheries-integrated-toolbox.github.io/resources/onboarding/github-account/)
+	3. We also have [open source guidelines](https://www.waterboards.ca.gov/resources/oima/cowi/open_source_code_handbook.html) from the college of water informatics which might be helpful.
+3. Data Ethical documentation
+	1. [Data ethics stuff: GovOps wrote a Medium article on Data Ethics](https://medium.com/caldata/practical-data-ethics-summary-of-what-were-learning-24d0e9d89cd0) 
+	2. [Fed ethics framework is one they mention](https://resources.data.gov/assets/documents/fds-data-ethics-framework.pdf) 
+	3. [FAIR and CARE principles resources are all over the place](https://www.nature.com/articles/s41597-021-00892-0)
+	4. [Here are another couple of things I found that is worth researching](https://directory.civictech.guide/listing/the-odis-data-ethics-canvas)
+	5. [CPTF eval toolkit](https://coloradolab.org/wp-content/uploads/2021/08/CPTF-Evaluation-Toolkit_June-2021_FINAL.pdf)
+4. GitHub usefule links an guidlines
+	1. [github skills](https://skills.github.com/)
+	2. [openscapes git clinic](https://openscapes.github.io/series/core-lessons/github/github-pub.html)
+	3. others?
+
```

Comment:
# Perhaps a section on Workflows in github with "bob stories" to describe them. 
 -  typical code workflow
 - a project document workflow
 -  a governance development workflow
 -  etc
A code workflow might be:
1. ID a feature or bug to address
2. create an issue 
3. Repo admin assigns to appropriate staff
4. Staff creates new branch to address issue in code
5. Staff completes fix and commits code (this step has its own workflow, simplified here)
6. Staff submits pull request
7. Repo admin assigns pull request review to designated Review Staff  
8. Review staff comment, approve, or deny fix (this step has its own workflow, simplified here; this step is iterative)
9. Fix is merged with default branch upon approval
10. Issue is closed; branch is closed

12.